phoenicis-javafx/src/main/java/org/phoenicis/javafx/views/mainwindow/MainWindowView.java

```diff
@@ -74,16 +74,22 @@ protected void setSideBar(SideBar sideBar) {
         this.leftContent.setContent(sideBar);
     }
 
-    public void showRightView(Node nodeToShow) {
+    public void setCenter(Node nodeToShow) {
         this.content.setCenter(nodeToShow);
     }
 
+    public void showDetailsView(Node nodeToShow) { this.content.setRight(nodeToShow); }
```

Comment:
Something else we should think about at this point is how to show a close button for the details view.
In PR #738 I've used a `StackPane` to add a close button. Alternatively we can use a `BorderPane` for example, this would add the close button at the top and not overlapping to the details view.

I'm not sure if we should address this in this PR but I think we can already lay down the foundation for this by adding a BorderPane or a StackPane.